consensus/core/src/authority_service.rs

```diff
@@ -94,7 +94,11 @@ impl<C: CoreThreadDispatcher> NetworkService for AuthorityService<C> {
                 .metrics
                 .node_metrics
                 .invalid_blocks
-                .with_label_values(&[peer_hostname, "handle_send_block"])
+                .with_label_values(&[
+                    peer_hostname,
+                    "handle_send_block",
+                    "BlockSignatureMissmatchSender",
```

Comment:
s/missmatch/mismatch

Reply:
Or `UnexpectedAuthority` as the error below. Signature is not verified here. This is only checking the peer against indicated author of the block.